app/forms/booking_request_form.rb

```diff
@@ -53,6 +53,13 @@ def step_two_valid?
     valid?
   end
 
+  def step_two_valid_excluding_eligibility?
+    @step = 2
+    valid?
+
+    (errors.keys - %i(appointment_type dc_pot)).empty?
```

Comment:
if we are going to do this can we extract `%i(appointment_type dc_pot)` into a constant.